tests/operators/ostream_test.cpp

```diff
@@ -0,0 +1,17 @@
+#include "../../src/bigint.hpp"
+
+int main() {
+	libbig::largeInt a;
+	std::cout << a << std::endl;
```

Comment:
if we write tests in this way, then what is the probability that written functions are behaving correctly? because we are not asserting the results with correct results anywhere in this test file. i think for stream operators a better way could be reading data from a `.txt` or `.dat` file and then asserting it with data provided in these `.txt` or `.dat` file, in this way if the assertions are not failing then we can say that things are going good.